README.md

```diff
@@ -309,6 +309,21 @@ git = "https://github.com/japaric/steed"
 stage = 2
 ```
 
+## Cargo mode
+
+Xaro supports configuring the mode in which `cargo` is invoked.
+The current options are `build` (the default), and `check`. This
+is used to determine which `cargo` subcommand will be invoked - either
+`build` or `check`.
+
+This is set via the `cargo_mode` entry in `Xargo.toml`:
+
+``` toml
+cargo_mode = "check"
```

Comment:
"mode" is an odd choice of word, I think cargo calls this a "command". Also since this only affects the sysroot, maybe something like `sysroot_cargo_command` or so would be better?